<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создать рассылку</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Параметр рассылки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Значение параметра</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inp">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="chB">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Имя</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Фамилия</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Провайдер почты</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номер телефона</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пол(М/Ж)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Всем</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="crtBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>560</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>530</y>
     <width>118</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите текст 
рассылки или 
выберите файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openTxt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выбрать файл
(txt)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="body">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>321</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="tm">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тема рассылки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="infBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>i</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
